src/main/java/sg/ncl/MainController.java

```diff
@@ -144,39 +144,95 @@ public String tools() {
         return "tools";
     }
 
-    @RequestMapping(value="/futureplan/download", method=RequestMethod.GET)
-    public void futureplanDownload(HttpServletResponse response) throws FuturePlanDownloadException, IOException {
+//    @RequestMapping("/resource2")
+//    public String resource2() {
+//        return "resource2";
+//    }
+
+    @RequestMapping("/tutorials")
+    public String tutorials() {
+        return "tutorials";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this method and declare a constant for this value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3400)